WiinUPro/Windows/MainWindow.xaml

```diff
@@ -59,7 +59,8 @@
                             <CheckBox x:Name="settingExclusiveMode" Content="Use Exclusive Mode" HorizontalAlignment="Left" Margin="10,50,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingExclusiveMode_Checked"/>

```

Comment:
Writing a tooltip for “Use Exclusive Mode” and “Use Toshiba Mode” might be useful, since these settings are not necessarily self-explanatory.

Reply:
Hi, I previously read here that Exclusive mode simply could not work on Win10, does re-enabling of this option mean it has been resolved as of 0.9.7 already?

Reply:
As far as I know it still doesn't work for the versions of Windows 10 it wasn't working on. I haven't verified that it still doesn't work on the latest version, but for those people who are earlier versions of Windows it is enabled.

Reply:
Thanks for clarifying! Speaking about tooltips, I note that for _Minimize to System_ Tray option, it says "Closing the application will instead minimize it to system tray". But it still actually closes and exit when I close [X] it, instead of minimizing to tray. 

Though it does minimize to tray when I click [-], where previously it stayed on my taskbar below.